scripts/functional-tests.sh

```diff
@@ -60,10 +61,12 @@ function init_test_script() {
 
     configdir_script=$(printf 'import platformdirs; print(platformdirs.user_config_dir("%s"))' "${PACKAGE_NAME}")
     datadir_script=$(printf 'import platformdirs; print(platformdirs.user_data_dir("%s"))' "${PACKAGE_NAME}")
+    downloaddir_script=$(printf 'import platformdirs; print(platformdirs.user_cache_dir("%s"))' "${PACKAGE_NAME}")
```

Comment:
nit: The name of the variable (and `ILAB_DOWNLOAD_DIR`) would be slightly more consistent with the rest if you'd replace `download` with `cache`. (we already have `user_data_dir` -> `datadir_script`, `user_config_dir` -> `configdir_script`...)

Reply:
Done!